assets/pages/digital-subscription-checkout/digitalSubscriptionCheckoutReducer.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { type CommonState } from 'helpers/page/page';
+
+import { type Action } from './digitalSubscriptionCheckoutActions';
+
+
+// ----- Types ----- //
+
+export type Stage = 'checkout' | 'thankyou';
+
+type PageState = {
+  stage: Stage;
+};
+
+export type State = {
+  common: CommonState,
+  page: PageState,
+};
+
+
+// ----- Reducer ----- //
+
+const initialState = {
+  stage: 'checkout',
+};
+
+function reducer(state: PageState = initialState, action: Action): PageState {
+
+  switch (action.type) {
+
+    case 'SET_STAGE':
```

Comment:
Is it possible to add tests for this reducer and action?

Reply:
Done.